ouroboros-consensus/test-consensus/Test/Consensus/ChainSyncClient.hs

```diff
@@ -370,13 +370,13 @@ updateClientState cfg chain ledgerState chainUpdates =
         where
           chain'       = foldl' (flip Chain.addBlock) chain bs
           ledgerState' = runValidate $
-            foldExtLedgerState cfg bs ledgerState
+            foldExtLedgerState False cfg bs ledgerState
       Nothing
       -- There was a roll back in the updates, so validate the chain from
       -- scratch
         | Just chain' <- Chain.applyChainUpdates chainUpdates chain
         -> let ledgerState' = runValidate $
-                 foldExtLedgerState cfg (Chain.toOldestFirst chain') testInitExtLedger
+                 foldExtLedgerState False cfg (Chain.toOldestFirst chain') testInitExtLedger
```

Comment:
Yeah, definitely want a new type with names. This is too opaque with just a mysterious `False` or `True`.